backstop/tests/tooltip.html

```diff
@@ -52,6 +52,11 @@
         margin-left: -40px;
         margin-top: 5px;
       }
+
+      section {
+        background-image: url('https://images.unsplash.com/photo-1544306094-e2dcf9479da3');
```

Comment:
I'm surprised this network image shows up in backstop tests 😮 @bentleyvk

Reply:
Machine has network so why not 😀 But it would be good to hide it in tests to avoid random failures.